src/Form/PostType.php

```diff
@@ -52,15 +52,18 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             ->add('title', null, [
                 'attr' => ['autofocus' => true],
                 'label' => 'label.title',
+                'empty_data' => '',
```

Comment:
[phpstan] this approach allows us to avoid nullable properties in the DTOs (forcing us to check for null before setting its value into a non-nullable argument) 

If you remove the `required` HTML attribute, the submitted form field data will be an empty string `''` instead of `null`, and `NotBlank` validator will do its job.